From 2679210f4ce804713bf1d244ac0fb8ac7b9b1e5f Mon Sep 17 00:00:00 2001
From: Nikolay Amiantov <ab@fmap.me>
Date: Thu, 25 Jul 2019 20:46:58 +0300
Subject: [PATCH 16/18] systemd-sleep: execute scripts in
 /etc/systemd/system-sleep

This is needed for NixOS to use such scripts as systemd directory is immutable.
---
 src/sleep/sleep.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/sleep/sleep.c b/src/sleep/sleep.c
index 7029352ca5..6d9c636872 100644
--- a/src/sleep/sleep.c
+++ b/src/sleep/sleep.c
@@ -182,6 +182,7 @@ static int execute(char **modes, char **states) {
         };
         static const char* const dirs[] = {
                 SYSTEM_SLEEP_PATH,
+                "/etc/systemd/system-sleep",
                 NULL
         };
 
-- 
2.27.0

